### PR TITLE
Prepare for bpmn-js@9

### LINKED ISSUE #64: When bumping to bpmn-js@v9.x.x, disable collapsed subprocess editing for cloud diagrams
__Is your feature request related to a problem? Please describe.__

With bpmn-js@9.x.x one will be able to edit contents inside collapsed subprocesses and then collapse them.
This collapsing will initially not be supported by the cloud stack (operate, zeebe). Reason is, that they have other priorities at the moment. They will ensure this works later.

Therefore we have to ensure that no BPMN diagrams with collapsed subprocesses with contents can be created for Cloud (not via the Camunda Modeler and also not via the Web IDE).

__Describe the solution you'd like__

When bumping to bpmn-js@v9.x.x, disable collapsed subprocess editing for cloud diagrams.

Note that will be re-enabled in the future (as soon as cloud gives us the go) 

__Describe alternatives you've considered__

Directly disable in Camunda Modeler and WebIDE.

__Additional context__

https://github.com/bpmn-io/internal-docs/issues/340

### PR DESCRIPTION
**Pull request by marstamm**
- enable drilldown for Platform Modeler
- disable drilldown for Cloud Modeler

closes #64

### CODE REVIEW

File: lib/base/Modeler.js [hunk 1]
```diff
@@ -9,10 +9,6 @@ import alignToOriginModule from '@bpmn-io/align-to-origin';
 
 import executableFixModule from 'bpmn-js-executable-fix';
 
-import signavioCompatModule from 'bpmn-js-signavio-compat';
```
Comment (pinussilvestrus) [inline lib/base/Modeler.js:12]: Don't we compat signavio anymore 🙁 ?

Comment (marstamm) [reply lib/base/Modeler.js:12]: We are still compatible with signavio, but don't need the extra module for it.
We now handle Subprocesses similarly to signavio and bpmn-js can expand them out of the box. Happy yo show you what we are doing :slightly_smiling_face:

Comment (marstamm) [reply lib/base/Modeler.js:12]: Collapse/Expand of multi-plane diagrams is now part of the core modeling experience

Comment (pinussilvestrus) [reply lib/base/Modeler.js:12]: Oh nice, I didn't know signavio compat was only about sub-processes collapsing 👍

Comment (philippfromme) [reply lib/base/Modeler.js:12]: Being able to remove the compatibility plugin is a good sign. 👍🏻

File: lib/camunda-cloud/Modeler.js [hunk 2]
```diff
@@ -48,6 +52,8 @@ inherits(Modeler, BaseModeler);
 Modeler.prototype._camundaCloudModules = [
   modelingModule,
   contextPadModule,
+  disableCollapsedSubprocessModule,
+  drilldownModule,
```
Comment (philippfromme) [inline lib/camunda-cloud/Modeler.js:56]: Why do we include the drilldown module in the first place if we disable it? Is it necessary?

Comment (marstamm) [reply lib/camunda-cloud/Modeler.js:56]: The drilldown module is shipped with the base modeler. The one we include as `_camundaCloudModules` overrides the default one with `null`

Comment (marstamm) [reply lib/camunda-cloud/Modeler.js:56]: Is there another way to disable modules? I could not find anything about it

Comment (philippfromme) [reply lib/camunda-cloud/Modeler.js:56]: Makes sense. Thanks for clarifying. Disabling modules using empty modules with the same key is fine. The only alternative would be monkey patching 🐒 .

**Review by philippfromme — APPROVED** [review]
✔️

**Pull request merged**